.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Code style
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
I think the latest version of this is >3

Reply:
Updated both steps v3